blog/_posts/2020-11-06-hybrid-cloud-with-inlets.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Hybrid Cloud with inlets
+description: Learn how to connect on-premises services and cloud services with inlets &reg;
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud 
+author_img: jsiebens
+date: 06-11-2020
+---
+
+Learn how to connect on-premises services and cloud services with inlets &reg;
+
+## Hybrid Cloud?
+
+When people talk about Hybrid Cloud, they talk about a way to combine a private cloud or on-premises service with one or more public cloud services. A hybrid cloud strategy can give a huge benefit for your business by moving workloads to a public cloud, leveraging the flexibility and robustness of managed services, while keeping sensitive data on a private cloud or local data center.
+
+In this post, we'll demonstrate how you can bring your on-premises services or databases into a Kubernetes cluster running on a public cloud.  
+This model applies for different use-cases:
+perhaps you are in the middle of a digital transformation where some parts of the architecture is deployed on a public cloud, but they still need to integrate with some legacy services
+you have some sensitive data to be kept in a private data center due to data residency regulation
+
+## Tutorial
+You will need an inlets PRO license, start a 14-day free trial.
+
+You'll need:
+- A Kubernetes cluster running on a public cloud (e.g. GKE, AKS, EKS, DOKS, ...)
+- `kubectl`, configured to connect to the cluster
+- A domain and access to your DNS admin panel to create a sub-domain
+- A service, like a database, running locally
+
+As an example, we will connect a WordPress instance running in the cloud with a MySQL server running locally. Still, this solution is perfectly applicable to other databases or services like e.g. an Oracle database, a MinIO cluster or a RabbitMQ service.
+
+![hybrid-mysql-wordpress](/images/2020-11-06-hybrid-cloud-with-inlets/mysql-wordpress.png)
+
+### Create the inlets-pro exit server
```

Comment:
For the tool / CLI use `inlets-pro` and for the product name can you use `inlets PRO`?